device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/Message.java

```diff
@@ -35,6 +35,16 @@
      */
     private String messageId;
 
+    /**
+     * Whether or not there is a list of nested messages included in this instance.
+     */
+    private boolean isBulk;
+
+    /**
+     * List of nested messages.
```

Comment:
what do nested messages mean here?

Reply:
I have moved it to batched messages. That makes the batch message a wrapper around the same object type of Message.